free-1.md

```diff
@@ -10,41 +10,83 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_My idea would help organize to do lists. Right now, people write to do 
+lists by what to do first, which is not always an easy way of thinking 
+about it and may cause people to forget tasks because they forgot
+about a certain category in their to do list. So, another way people think of
```

Comment:
I use a lot of to-do lists and I definitely have the problem you just described: when I try to categorize tasks into multiple to-do lists, I forget the existence of a list; but when I lump all my tasks into a single list, it's hard to prioritize and organize the massive list.